docs/faq/admin.txt

```diff
@@ -109,3 +109,12 @@ Android and iOS.
 
 Depending on feature support, there *may* be minor stylistic differences
 between browsers. These are considered acceptable variations in rendering.
+
+What assistive technologies are supported for using the admin?
+==============================================================
+
+We want the admin to be compatible with wide range of assistive
+technologies, but are aware of many blockers currently. We intend to support
+all latest versions of major assistive technologies, including
+JAWS, NVDA, VoiceOver iOS, TalkBack, Dragon, VoiceOver macOS, Voice Control,
+ZoomText, Windows Contrast Themes, and screen magnifiers.
```

Comment:
```suggestion
ZoomText, Windows Contrast Themes and screen magnifiers.
```

Just FYI commas before "and" is only correct when joining independent clauses.

Reply:
Doesn’t the removal of the comma make it sound like screen magnifiers are only a Windows thing?

Reply:
Actually you're originally right, looks like _not_ using the "serial comma" is a thing in my country (and I was wrong about it "not being correct" as it's still acceptable) but it's common in US english. 😅

It's better to use it, as they say in The Zen of Python: "Explicit is better than implicit." 👍